packages/flutter_tools/lib/src/flutter_project_metadata.dart

```diff
@@ -141,16 +141,27 @@ class FlutterProjectMetadata {
   final Logger _logger;
 
   final File _metadataFile;
+  File get file => _metadataFile;
```

Comment:
should `_metadataFile` just be final public then?

Reply:
Yeah, I guess so!